### PR TITLE
Fix newlib build issues

### PR DESCRIPTION
**Pull request by Ghabry**
the typical stuff...

### CODE REVIEW

**Comment by Ghabry** [comment]
This is fine when the 3DS PR build passes:
https://ci.easyrpg.org/view/Pull%20Requests/job/player-3ds-pr/

**Comment by Ghabry** [comment]
3ds build successful

**Review by fdelapena — APPROVED** [review]

File: src/game_system.cpp [hunk 1]
```diff
@@ -150,12 +150,12 @@ void Game_System::SePlay(const lcf::rpg::Sound& se, bool stop_sounds) {
 	// Validate
 	if (se.volume < 0 || se.volume > 100) {
 		Output::Debug("SE {} has invalid volume {}", se.name, se.volume);
-		volume = Utils::Clamp(se.volume, 0, 100);
+		volume = Utils::Clamp<uint32_t>(se.volume, 0, 100);
```
Comment (carstene1ns) [inline src/game_system.cpp:153]: Guess this should be `int32_t` for consistency.

Comment (Ghabry) [reply src/game_system.cpp:153]: yeah indeed. ups ^^

**Review by fmatthew5876 — APPROVED** [review]

**Pull request merged**